server/index.js

```diff
@@ -8,7 +8,7 @@ var parser = require('body-parser');
 const cors = require('cors');
 
 // router
-var router = require('./routes.js');
+var router = require('./newroutes.js');
 
 // run express server
 var app = express();
```

Comment:
Use const declaration.

Reply:
Legacy code. Asked Leslie for clarification on whether these types of changes should be made or left alone when making updates.